cluster-up/cluster/kind-k8s-sriov-1.17.0/sriov-components/sriov_components.sh

```diff
@@ -107,7 +107,7 @@ function sriov_components::wait_allocatable_resource() {
 
 function sriov_components::deploy_multus() {
   echo 'Deploying Multus'
-  sed "s#nfvpe/multus#quay.io/kubevirtci/multus#" "$MULTUS_MANIFEST" | _kubectl apply -f -
```

Comment:
Leftovers from a rebase that went not correct, please check it

Reply:
Removed.